editor/src/components/canvas/canvas-strategies/interaction-state.ts

```diff
@@ -347,6 +352,7 @@ function updateInteractionDataViaMouse(
             _accumulatedMovement: zeroCanvasPoint,
             spacePressed: false,
             zeroDragPermitted: currentData.zeroDragPermitted,
+            hasBeenPastThreshold: false,
```

Comment:
Actually `drag` is null until it exceeded the threshold (but once it did exceed it, it stays not-null even if you go back near zero magnitude), so you can just nullcheck that. there is also the unfortunately named `isNotYetStartedDragInteraction` helper that relies on the null value